### ISSUE TITLE
.NET 4.6 compatibility

### ISSUE THREAD

**Issue by hashitha** [EdonGashi/WpfMaterialForms#32]
I am trying to build the latest code but failing. Is there an alternative to AttributeBuilder

AttributeBuilder.Standard 1.3.27 is not compatible with net461 (.NETFramework,Version=v4.6.1) / win-x64. Package AttributeBuilder.Standard 1.3.27 supports: netstandard2.0 (.NETStandard,Version=v2.0)			0

**Comment by mvmcjr** [comment]
@hashitha Oops, this is actually my fault, will fix it in a bit

**Comment by mvmcjr** [comment]
@hashitha Hmm thats weird, [according to the Microsoft .NET Standard page](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) it should be compatible with .NET Framework 4.6.1. Do you have the .NET Core sdk installed?

**Comment by hashitha** [comment]
@redbaty yes I have .NET core installed. I am using VS 2017. I am getting error in Proxier

![image](https://user-images.githubusercontent.com/10569142/35108774-616ecab6-fcaf-11e7-9bc4-cf9cfe289cb7.png)

**Comment by mvmcjr** [comment]
@hashitha Hmm anyway, I'll publish a version of Proxier that supports .NET 4.6.1. In the meantime if you are in a hurry you could try to downgrade your WPFMaterialForms package version or move to .NET 4.7

**Comment by hashitha** [comment]
@redbaty no worries take your time

**Comment by redbaty** [comment]
@hashitha By the way, have you checked out the new lib? It supports the progress stuff that you mentioned here: #23

**Comment by hashitha** [comment]
@redbaty yes that is what I wanted to have a look

**Comment by mvmcjr** [comment]
@hashitha Actually my bad, I haven't pushed out the new progress stuff yet, I'll try to add it later. But it should be more stable since the Mapper stuff have been moved out to a separated project

**Comment by mvmcjr** [comment]
@hashitha Try it with the newest package, it is still validating but should be available sometime soon

**Comment by hashitha** [comment]
@redbaty now getting a different error. I tried cloning the repo again as well.

![image](https://user-images.githubusercontent.com/10569142/35126384-260331ea-fce8-11e7-84c8-d2bc5730c699.png)

**Comment by mvmcjr** [comment]
@hashitha Hmm that shouldn't happen, but anyway this should be a easy fix, try to install [this package](https://www.nuget.org/packages/System.ValueTuple/) if it doesn't help try to install System.Runtime aswell

**Comment by hashitha** [comment]
@redbaty I cant install any packages gives this error.

![image](https://user-images.githubusercontent.com/10569142/35127693-44e5153c-fcee-11e7-9879-e9244b8bb648.png)

![image](https://user-images.githubusercontent.com/10569142/35127797-87a73fee-fcee-11e7-82a7-0549e4c58efe.png)

**Comment by edongashi** [comment]
This is a ridiculous number of dependencies. We are migrating this library to a module based structure. Once that is stable things will become much cleaner.
